qutebrowser/browser/pdfjs.py

```diff
@@ -202,15 +203,30 @@ def _read_from_system(system_path, names):
     return (None, None)
 
 
+def _get_pdfjs_basename():
+    """Checks for pdf.js module availability and returns the basename."""
+    final_ext = None
+    exts = ['pdf.js', 'pdf.mjs']
+    for ext in exts:
+        try:
+            final_ext = ext
+            get_pdfjs_res('build/' + ext)
+        except PDFJSNotFound:
+            final_ext = None
+    if not final_ext:
+        raise PDFJSNotFound('build/pdf.js') from None
```

Comment:
I find this logic quite hard to follow. Something like

```suggestion
        try:
            get_pdfjs_res(f'build/{ext}')
            return ext
        except PDFJSNotFound:
            pass

    raise PDFJSNotFound('build/pdf.js')
```

seems to make things a bit simpler. I also removed the `from None` - you're not in an `except ...:` there, so it does nothing.

Reply:
duly noted and simplified